weblate/addons/models.py

```diff
@@ -48,24 +48,32 @@
 
 
 class AddonQuerySet(models.QuerySet):
-    def filter_component(self, component):
-        return self.prefetch_related("event_set").filter(
-            (Q(component=component) & Q(project_scope=False))
-            | (Q(component__project=component.project) & Q(project_scope=True))
-            | (Q(component__linked_component=component) & Q(repo_scope=True))
-            | (Q(component=component.linked_component) & Q(repo_scope=True))
-        )
+    def filter_component(self, component=None, project=None):
```

Comment:
This is called filter_**component** so let's stick with filtering components only. Please add a separate method for filtering projects instead.